include/grpc/impl/compression_types.h

```diff
@@ -41,7 +41,11 @@ extern "C" {
  * Its value is an int from the \a grpc_compression_algorithm enum. */
 #define GRPC_COMPRESSION_CHANNEL_DEFAULT_ALGORITHM \
   "grpc.default_compression_algorithm"
-/** Default compression level for the channel.
+/** Set the Default compression level for the channel.
```

Comment:
I think we should just say - 
"Set the default compression level for the channel. Valid values are defined by the enum type `grpc_compression_level`, defaults to `GRPC_COMPRESS_LEVEL_NONE`."